packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -165,29 +251,95 @@ export interface TreeView<TSchema extends ImplicitFieldSchema> extends IDisposab
 	 */
 	upgradeSchema(): void;
 
+	/**
+	 * Initialize the tree, setting the stored schema to match this view's schema and setting the tree content.
+	 *
+	 * Only valid to call when this view's {@link SchemaCompatibilityStatus.canInitialize} is true.
+	 *
+	 * Applications should typically call this function before attaching a `SharedTree`.
+	 * @param content - The content to initialize the tree with.
+	 */
+	initialize(content: InsertableTreeFieldFromImplicitField<TSchema>): void;
+
 	/**
 	 * Events for the tree.
 	 */
 	readonly events: Listenable<TreeViewEvents>;
 }
 
 /**
- * Information about how a view schema was incompatible.
+ * Information about a view schema's compatibility with the document's stored schema.
+ *
+ * See SharedTree's README for more information about choosing a compatibility policy.
  * @public
  */
-export interface SchemaIncompatible {
+export interface SchemaCompatibilityStatus {
 	/**
-	 * True iff the view schema supports all possible documents permitted by the stored schema.
+	 * Whether the view schema allows exactly the same set of documents as the stored schema.
+	 *
+	 * @remarks
+	 * Equivalence here is defined in terms of allowed documents because there are some degenerate cases where schemas are not
+	 * equal in a strict sense but still allow the same documents, and the document notion is more useful to applications.
+	 *
+	 * Examples which are expressible where this may occur include:
+	 * - schema repository `A` has extra schema which schema `B` doesn't have, but they are unused (i.e. not reachable from the root schema)
+	 * - field in schema `A` has allowed field members which the corresponding field in schema `B` does not have, but those types are not constructible (ex: an object node type containing a required field with no allowed types)
+	 *
+	 * These cases are typically not interesting to applications.
+	 */
+	readonly isExactMatch: boolean;
+
+	/**
+	 * Whether the current view schema is sufficiently compatible with the stored schema to allow viewing tree data.
+	 * If false, {@link TreeView.root} will throw upon access.
+	 *
+	 * A necessary condition for this to be true is that the documents allowed by the view schema are a subset of those allowed by the stored schema.
```

Comment:
I find this confusing.

Is this talking about the set of schema compatible documents being a subset? If so, that seems backwards (you can view things if you view schema allows all things you might want to view, not if it allows a subset of them).

If this is instead talking about tree content, then I think it would need to be more detailed. For example if the view schem permits empty (optional root) then is it compatible with everything, since empty is a structural subset of all possible trees of all valid schema?

Reply:
I agree with your other comment that this bit is more appropriate for privateRemarks though, so this bit perhaps doesn't matter as much